.laminas-ci.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ignore_php_platform_requirements": {
+    "8.2": true
```

Comment:
AFAIK, all dependencies in `composer.json` should be compatible with PHP 8.2, at this stage?

Can you please try dropping this?